version/version.go

```diff
@@ -4,11 +4,10 @@ import (
 	"fmt"
 	"runtime"
 
-	sdkVersion "github.com/operator-framework/operator-sdk/version"
-	logf "sigs.k8s.io/controller-runtime/pkg/log"
+	ctrl "sigs.k8s.io/controller-runtime"
 )
 
-var log = logf.Log.WithName("version")
+var log = ctrl.Log.WithName("version")
```

Comment:
Is there a reason for this to be global?

Reply:
Not that I know of